src/ansys/fluent/core/logging.py

```diff
@@ -25,6 +25,24 @@ def root_config():
         logger.addHandler(ch)
 
 
+def set_console_loggers_level(level: Union[str, int]):
```

Comment:
Minor suggestion, slightly clearer to me:
```suggestion
def set_console_logging_level(level: Union[str, int]):
```
We are setting the level of logging calls that output to console, the actual loggers (which propagate to console through the root pyfluent logger) will still be set to different levels